modules/cats/src/main/tut/README.md

```diff
@@ -0,0 +1,41 @@
+# Cats module for PureConfig
+
+Adds support for selected [cats](http://typelevel.org/cats/) classes to PureConfig.
+
+## Add pureconfig-cats to your project
+
+In addition to [core pureconfig](https://github.com/melrief/pureconfig), you'll need:
+
+```scala
+libraryDependencies += "com.github.pureconfig" %% "pureconfig-cats" % "0.7.1"
+```
+
+## Example
+
+To load a `NonEmptyList[Int]` into a configuration, we need a class to hold our configuration:
```

Comment:
My issue is not so much with using a case class to hold a config (which is pretty much mandatory) as with the particular example and wording being used. Most modules make the detail I mentioned clearer because the holding case classes in their examples have more than one field than one field and they don't imply that "we need a class" to "load a `NonEmptyList[Int]` into a configuration". If you see `joda`, for example, they mention an "object to receive our configuration" with two `DateTime`s and talk about reading "a GreatDatesConfig", not a `DateTime` directly. At least for me, it makes it clearer that those non-root classes like `DateTime` can be used normally in case class fields.

I don't know if I explained myself clearly. I'm also aware I'm being pedantic here, so please take this as a suggestion and ignore it if you don't think this is an issue.